static/app/views/eventsV2/table/columnEditModal.spec.jsx

```diff
@@ -149,200 +171,308 @@ describe('EventsV2 -> ColumnEditModal', function () {
-        'transaction.duration'
+    it('shows additional inputs for multi-parameter functions', async function () {
+      mountModal(
+        {
+          columns: [columns[0]],
+          onApply,
+        },
+        initialData
       );
+      await selectByLabel('percentile(\u2026)', {
+        name: 'field',
+        at: 0,
+      });
 
-      // Input should show and have default value.
-      const refinement = wrapper.find('QueryField input[inputMode="numeric"]');
-      expect(refinement.props().value).toBe('0.5');
+      expect(screen.getAllByTestId('label')[0]).toHaveTextContent('percentile(…)');
+      expect(
+        within(screen.getByTestId('queryField')).getByDisplayValue(0.5)
+      ).toBeInTheDocument();
     });
 
-    it('handles scalar field parameters', function () {
-      selectByLabel(wrapper, 'apdex(\u2026)', {name: 'field', at: 0, control: true});
-
-      // Parameter select should display and use the default value.
-      const field = wrapper.find('QueryField input[name="refinement"]');
-      expect(field.props().value).toBe('300');
+    it('handles scalar field parameters', async function () {
+      mountModal(
+        {
+          columns: [columns[0]],
+          onApply,
+        },
+        initialData
+      );
+      await selectByLabel('apdex(\u2026)', {
+        name: 'field',
+        at: 0,
+      });
 
-      // Trigger a blur and make sure the column is not wrong.
-      field.simulate('blur');
+      const input = screen.getAllByRole('textbox')[1];
+      expect(input).toHaveValue('300');
+      userEvent.click(screen.getByRole('button', {name: 'Apply'}));
 
-      // Apply the changes so we can see the new columns.
-      wrapper.find('Button[priority="primary"]').simulate('click');
-      expect(onApply).toHaveBeenCalledWith([
-        {kind: 'function', function: ['apdex', '300', undefined, undefined]},
-      ]);
+      await waitFor(() => {
+        expect(onApply).toHaveBeenCalledWith([
+          {kind: 'function', function: ['apdex', '300', undefined, undefined]},
+        ]);
+      });
     });
 
-    it('handles parameter overrides', function () {
-      selectByLabel(wrapper, 'apdex(\u2026)', {name: 'field', at: 0, control: true});
-
-      // Parameter select should display and use the default value.
-      const field = wrapper.find('QueryField input[name="refinement"]');
-      expect(field.props().value).toBe('300');
-      expect(field.prop('placeholder')).toBe(undefined);
+    it('handles parameter overrides', async function () {
+      mountModal(
+        {
+          columns: [columns[0]],
+          onApply,
+        },
+        initialData
+      );
+      await selectByLabel('apdex(…)', {
+        name: 'field',
+        at: 0,
+      });
 
-      // Trigger a blur and make sure the column is not wrong.
-      field.simulate('blur');
+      const input = screen.getAllByRole('textbox')[1];
+      expect(input).toHaveValue('300');
```

Comment:
```suggestion
      expect(screen.getAllByRole('textbox')[1]).toHaveValue('300');
```

can we check for the name of the `input`, does it contain an `aria-label`?